hrm-service/service-tests/db-init/migrate.js

```diff
@@ -0,0 +1,46 @@
+// eslint-disable-next-line global-require,import/no-extraneous-dependencies
```

Comment:
Is this testing that all migrations run under 2 seconds? Or I'm missing something else?